color.js

```diff
@@ -12,11 +12,23 @@ export class Color {
   }
 
```

Comment:
It's a great job - you've done really well! I just have some little changes to suggest